src/options/smt_options.toml

```diff
@@ -11,6 +11,16 @@ header = "options/smt_options.h"
   read_only  = true
   help       = "dump preprocessed assertions, etc., see --dump=help"
 
+[[option]]
```

Comment:
Can you call this dtPoliteOptimize dt-polite-optimize?  Also, this should be in datatype_options.

Reply:
done. also made it default to true.